src/tests/unit_ecdsa.cpp

```diff
@@ -106,6 +106,7 @@ Test::Result test_decode_ver_link_SHA256()
    return result;
    }
 
+#if defined(BOTAN_HAS_SHA1)
 Test::Result test_decode_ver_link_SHA1()
```

Comment:
Instead of compiling this test out have it still load the certificates, then check that if SHA1 is disabled the verification fails. That (albeit somewhat implicitly) tests that we can still parse a SHA-1 signed certificate even if we cannot verify it.

Reply:
Changed